vizro-core/src/vizro/models/_navigation/nav_link.py

```diff
@@ -58,10 +59,14 @@ def build(self, *, active_page_id=None):
 
         button = dbc.Button(
             [
-                html.Span(self.icon, className="material-symbols-outlined"),
-                # TODO: commented out until we insert styling for the tooltip or find a better way to display it (e.g.
-                # try dbc.Popover or Dash mantine components tooltip?).
-                # dbc.Tooltip(html.P(self.label), target=self.id, placement="bottom", className="custom-tooltip"),
+                dmc.Tooltip(
+                    label=self.label,
+                    offset=6,
+                    withArrow=True,
+                    zIndex=1,
+                    children=[html.Span(self.icon, className="material-symbols-outlined")],
+                    position="bottom-start",
+                )
```

Comment:
Just curious: Is the Tooltip always presented, without any option to remove it?

Reply:
This is actually a very good question - people should be able to remove it if they want. Currently, if you provide an empty string, the tooltip is still visible. Shouldn't the `label` be Optional @antonymilne @nadijagraca ? I understand that for the horizontal alignment it should be a mandatory field, but for the vertical alignment it should be Optional?

Reply:
I think we should leave this compulsory:
* it will be easy (non-breaking) in future to make it optional, but the reverse direction is not true
* as you say, when it comes to a horizontal navbar, it would be compulsory, and having something optional in one layout and not the other could be tricky and confusing
* always requiring a tooltip is arguably important for accessibility and good design principles (?)
* in reality, much of the time this label is automatically populated anyway (from a higher level `pages` field) and so the user does not need to explicitly specify it anyway

The intention here is to keep things simple and non-committal since we don't actually know much about how people will use the components in reality (or even whether they will use them at all). We can always change in future after we've seen real world usage and identified what common requirements/complaints/etc. are. e.g. maybe we'll want a whole new field `tooltip` that's distinct from `label`, maybe not; maybe we will make it so that an empty string or other `False`y value for `label` doesn't render a tooltip. But unless you think these are important requirements to enable upfront, I would not do so yet.

Reply:
Got it! No, all good - I am fine with not enabling the possibility of disabling the tooltip until people request it